src/main/java/lotto/domain/LastWeekLottoValidator.java

```diff
@@ -57,14 +65,14 @@ private boolean isSizeCorrect(String[] number) {
      * 각 번호는 1 ~ 45사이의 수 입니다.
      * 6개의 번호가 이를 만족하면 true, 아니면 false 를 반환합니다.
      *
-     * @param number 지난 주 당첨 번호
+     * @param numbers 지난 주 당첨 번호
      * @return boolean
      * */
-    private boolean isNumberCorrect(String[] number) {
+    private boolean isNumberCorrect(String[] numbers) {
         return Arrays
-                .stream(number)
+                .stream(numbers)
                 .mapToInt(Integer::parseInt)
-                .allMatch(n -> n >= MIN_LOTTO_NUMBER && n <= MAX_LOTTO_NUMBER);
+                .allMatch(number -> number >= MIN_LOTTO_NUMBER && number <= MAX_LOTTO_NUMBER);
```

Comment:
개인적으로 스트림안의 조건은 메서드로 분리하는 편입니다.
의도를 명확하게 할 수 있어요 
```suggestion
                .allMatch(this::isValidateNumber);


......            
      private boolean isValidateNumber(int number) {
      return number >= MIN_LOTTO_NUMBER && number <= MAX_LOTTO_NUMBER;
   }
```

Reply:
```
public void winningNumbersValidation(List<Integer> numbers) {
    // 숫자 개수 검사
    this.checkCorrectLottoFormat(numbers);

    // 범위 검사
    numbers.forEach(this::checkCorrectRange);

    // 중복 검사
    this.checkUniqueNumber(numbers);
}
```

이렇게 변경해보았어요 !